src/Mod/Sketcher/Gui/CommandSketcherBSpline.cpp

```diff
@@ -1040,6 +1040,7 @@ void CmdSketcherInsertKnot::activated(int iMsg)
         QInputDialog knotDialog(Gui::getMainWindow());
         knotDialog.setInputMode(QInputDialog::DoubleInput);
         knotDialog.setDoubleDecimals(8);
```

Comment:
This is a mistake. It must be set to the user-specified number of decimals.

Search in The Part/Gui code for "XDirectionEdit" so see how it is done there.

Reply:
Thanks for the pointer. Done.

Reply:
> Done.

I don't see a new commit from you.

Reply:
I meant in my local machine :). This was a minor edit so i didn't expect it to take this long to compile and test.

Reply:
A rebase to latest master helps me in these cases. Then when I switch between the rebased branches, I really only need to recompile the files changed in the PR.

In case you use MSVC, I recently found this new CMake flag:
FREECAD_USE_MP_COMPILE_FLAG
(listed here: https://wiki.freecadweb.org/Compile_on_Windows#Options_for_the_build_process
 @chennes , many thanks for adding this info, this sped up my workflow a lot)

Reply:
I don't use MSVC but thanks for letting me know. I do use `make -j$(nproc --ignore=2)` on Linux.

Reply:
I guess I must have rebased to latest master recently but the already compiled version was not rebased so that caused everything to be compiled? I'm not really sure why this happens.